io.github.mfat.jottr.yml

```diff
@@ -0,0 +1,92 @@
+app-id: io.github.mfat.jottr
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.7'
+command: jottr
+separate-locales: false
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --socket=pulseaudio
+  - --env=FLATPAK_ENABLE_SDK_EXT=*
+  - --env=QTWEBENGINE_DICTIONARIES_PATH=/app/qtwebengine_dictionaries
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --env=PYTHONPATH=/app/lib/python3.11/site-packages:/app/lib/python3/site-packages:/app/lib/python3.11/dist-packages
+modules:
+  - name: bundled-fonts
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/fonts
+      - cp -r dejavu/* /app/share/fonts/
+      - cp -r droid/* /app/share/fonts/
+      - cp -r firacode/* /app/share/fonts/
+      - cp -r jetbrains-mono/* /app/share/fonts/
+      - cp -r liberation/* /app/share/fonts/
+      - cp -r noto/* /app/share/fonts/
+      - fc-cache -fv /app/share/fonts
+    sources:
+      - type: dir
+        path: fonts
+        dest: .
+
+  - name: python-dependencies
+    buildsystem: simple
+    build-options:
+      build-args: [ "--share=network" ]
+      env:
+        QTWEBENGINE_DISABLE_SANDBOX: '1'
+        PYTHONPATH: /app/lib/python3.11/site-packages:/app/lib/python3/site-packages:/app/lib/python3.11/dist-packages
+    build-commands:
+      - mkdir -p /app/lib/python3.11/site-packages
+      - mkdir -p /app/lib/python3.11/dist-packages
+      - pip3 install --no-cache-dir --prefix=/app PyQt5==5.15.11 PyQtWebEngine==5.15.7
```

Comment:
not sure why you have 6.7 then again trying to install 5.15

Reply:
this is also in the pyqt baseapp https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp